### PR TITLE
New issue templates

### PR DESCRIPTION
**Pull request by runleonarun**
closes https://github.com/dbt-labs/product-docs-team/issues/19
Adding new issue templates.

### CODE REVIEW

File: .github/ISSUE_TEMPLATE/config.yml [hunk 1]
```diff
@@ -0,0 +1,5 @@
+blank_issues_enabled: true
+contact_links:
+  - name: dbt Labs Community
+    url: https://getdbt.slack.com/archives/CBSQTAPLG
```
Comment (jasnonaz) [inline .github/ISSUE_TEMPLATE/config.yml]: Do we want to point them towards the signup page? https://www.getdbt.com/community/join-the-community

File: .github/ISSUE_TEMPLATE/improve-docs.yml [hunk 2]
```diff
@@ -0,0 +1,42 @@
+name: Improve content
+description: Make a suggestion to improve the content in an existing article.
```
Comment (jasnonaz) [inline .github/ISSUE_TEMPLATE/improve-docs.yml]: Small thing but do we call them `articles` or `pages`? I've normally seen `page` for docs site.

File: .github/ISSUE_TEMPLATE/improve-the-site.yml [hunk 3]
```diff
@@ -0,0 +1,42 @@
...
+    attributes:
+      label: Contributions
+      description: Please read the contribution docs before opening an issue or pull request.
+      options:
+        - label: I have read the contribution docs, and understand what's expected of me. 
+
+  - type: textarea
+    attributes:
+      label: What article(s) or areas on docs.getdbt.com are affected?
+      description: Include links to articles where you're seeing a problem, screenshots, what browser you're using, etc.
```
Comment (jasnonaz) [inline .github/ISSUE_TEMPLATE/improve-the-site.yml]: Is this just for reporting a problem or might they also be making a suggestion about change?

**Review by jasnonaz — APPROVED** [review]
Fantastic changes! Thank you for putting this together.

File: .github/ISSUE_TEMPLATE/config.yml (hunk 1 above)
Comment (runleonarun) [inline .github/ISSUE_TEMPLATE/config.yml]: Great point, @jasnonaz!
```suggestion
    url: https://www.getdbt.com/community/join-the-community
```

File: .github/ISSUE_TEMPLATE/improve-docs.yml (hunk 2 above)
Comment (runleonarun) [reply .github/ISSUE_TEMPLATE/improve-docs.yml]: I can get behind using "page!"
```suggestion
description: Make a suggestion to improve the content in an existing page.
```

File: .github/ISSUE_TEMPLATE/improve-the-site.yml [hunk 4]
```diff
@@ -0,0 +1,42 @@
...
+    id: contributions
+    attributes:
+      label: Contributions
+      description: Please read the contribution docs before opening an issue or pull request.
+      options:
+        - label: I have read the contribution docs, and understand what's expected of me. 
+
+  - type: textarea
+    attributes:
+      label: What article(s) or areas on docs.getdbt.com are affected?
```
Comment (runleonarun) [inline .github/ISSUE_TEMPLATE/improve-the-site.yml]:
```suggestion
      label: What page(s) or areas on docs.getdbt.com are affected?
```

File: .github/ISSUE_TEMPLATE/improve-the-site.yml (hunk 3 above)
Comment (runleonarun) [inline .github/ISSUE_TEMPLATE/improve-the-site.yml]:
```suggestion
      description: Include links to pages where you're seeing a problem or making a suggestion. Also include screenshots, what browser you're using, etc.
```

Comment (runleonarun) [reply .github/ISSUE_TEMPLATE/improve-the-site.yml]: I like the idea of keeping it open to either!

File: .github/ISSUE_TEMPLATE/improve-docs.yml [hunk 5]
```diff
@@ -0,0 +1,42 @@
...
+    id: contributions
+    attributes:
+      label: Contributions
+      description: Please read the contribution docs before opening an issue or pull request.
+      options:
+        - label: I have read the contribution docs, and understand what's expected of me. 
+
+  - type: textarea
+    attributes:
+      label: What article on docs.getdbt.com is affected?
```
Comment (runleonarun) [inline .github/ISSUE_TEMPLATE/improve-docs.yml]:
```suggestion
      label: What page on docs.getdbt.com is affected?
```

File: .github/ISSUE_TEMPLATE/improve-docs.yml [hunk 6]
```diff
@@ -0,0 +1,42 @@
...
+      description: Please link to the article you'd like to see updated.
```
Comment (runleonarun) [inline .github/ISSUE_TEMPLATE/improve-docs.yml]:
```suggestion
      description: Please link to the page you'd like to see updated.
```

File: .github/ISSUE_TEMPLATE/improve-docs.yml [hunk 7]
```diff
@@ -0,0 +1,42 @@
...
+    validations:
+      required: true
+
+  - type: textarea
+    attributes:
+      label: What part(s) of the article would you like to see updated?
```
Comment (runleonarun) [inline .github/ISSUE_TEMPLATE/improve-docs.yml]:
```suggestion
      label: What part(s) of the page would you like to see updated?
```

**Pull request merged**